operator/docker/base/Dockerfile

```diff
@@ -45,15 +55,65 @@ RUN MO_VERSION=${VERSION_STAMP} mage operator:buildGo
 
 # Second stage
 # Produce a smaller image than the one used to build the code
-FROM alpine:3.9
-
+FROM alpine:3.9 as cass-operator
 ENV GOPATH=/go
 
 RUN mkdir -p /var/lib/cass-operator/
 RUN touch /var/lib/cass-operator/base_os
 WORKDIR /go
 
 # All we need from the builder image is operator executable
-COPY --from=builder /cass-operator/build/bin/cass-operator-linux-amd64 bin/operator
+ARG TARGETARCH
+COPY --from=builder /cass-operator/build/bin/cass-operator-linux-${TARGETARCH} bin/operator
 
 CMD [ "/go/bin/operator" ]
+
+# UBI Image
```

Comment:
took me a second to figure out what happened, but making one Dockerfile and using the targets is good with me